### PR TITLE
W-16325926 Adding steps for each type of connected app

### PR DESCRIPTION
**Pull request by valkyrie69**
# Writer's Quality Checklist

Before merging your PR, did you:

- [ ] Run spell checker
- [ ] Run link checker to check for broken xrefs
- [ ] Check for orphan files
- [ ] Perform a local build and do a final visual check of your content, including checking for:
  - Broken images
  - Dead links
  - Correct rendering of partials if they are used in your content
  - Formatting issues, such as:
    - Misnumbered ordered lists (steps) or incorrectly nested unordered lists
    - Messed up tables
    - Proper indentation
    - Correct header levels
- [ ] Receive final review and signoff from:
  - Technical SME
  - Product Manager
  - Editor or peer reviewer
  - Reporter, if this content is in response to a reported issue (internal or external feedback)
- [ ] If applicable, verify that the software actually got released

### CODE REVIEW

File: modules/ROOT/pages/connected-apps-end-users.adoc
Comment (dlhforce) [inline modules/ROOT/pages/connected-apps-end-users.adoc:1]: This doc is targeted to end users, most of which would not have permissions to follow the administrative instructions added here.

It seems like the Connected Apps for Developers doc would be a better fit. Indeed, much of this new information is already there, just organized differently.

https://github.com/mulesoft/docs-access-management/blob/6f4f9e10b1fcf86738ab74fffe3d6cabc59c2ae8/modules/ROOT/pages/connected-apps-developers.adoc?plain=1#L29

File: modules/ROOT/pages/connected-apps-developers.adoc [hunk 1]
```diff
@@ -8,67 +8,71 @@ In addition to enabling new flows to authenticate users, developers can enable o
...
 
 include::partial$include-nav-connected-apps.adoc[]
 
-. In the *Owned Apps* section, click *Create App*.
-. In the *Create App* page, complete the following fields:
-.. *Name* +
-A unique name for the application
+. In *Owned Apps*, click *Create App*.
+. In the *Create App* page, enter a unique name for the application.
+. For *Type*, select *App acts on behalf of a user*.
```
Comment (dlhforce) [inline modules/ROOT/pages/connected-apps-developers.adoc]: First, I *love* that you're splitting out separate sections for "on Its Own Behalf" vs "on Behalf of a User". It is so much more readable!

I do think, though, that the first few steps should be consistent between the two sections, since the flow doesn't diverge until you select the grant type. In particular, it feels strange that there are different starting points between the different sets of steps; the user app steps here assume you're already in Connected Apps, whereas the client app steps below don't even assume that you're in Access Management yet.

If nothing else, I think it makes a lot more sense to steer the user towards "Connected Apps" as below, instead of steering them to "Owned Apps". If I was simply told "In Owned Apps, click Create App," I think it would take me awhile to figure out that Owned Apps was a tab label _within_ the Connected Apps page, and not a top-level navigation option. Up to you whether or not it's worth mentioning Owned Apps at all, I just think it should be contexualized if mentioned.

Also, it's probably worth at least mentioning that a Business Group can be selected before creating an app, although I presume it will be left untouched in the majority of cases, to simply default to the root org (or whatever org a non-root-org admin can access).

File: modules/ROOT/pages/connected-apps-developers.adoc [hunk 2]
```diff
@@ -112,27 +117,28 @@ The access token can now be used to call Anypoint Platform APIs on the user’s
...
+
+To request a new access token, send a request to the token endpoint using the `refresh_token` grant type. When authentication is successful, a refresh token is exchanged for an access token.
 
 === Password
 
-The `password` grant type is used when the application exchanges the user’s username and password for an access token.
-Because the app has to collect the user’s credentials and its limitation with no mechanisms for things like multi-factor authentication, it is not recommended to use this grant type.
+Use the `password` grant type when the application exchanges the user’s username and password for an access token.
+
+Because the app has to collect the user’s credentials and the `password` grant type's limitation in using mechanisms for things like multi-factor authentication, using this grant type isn't recommended.
```
Comment (dlhforce) [inline modules/ROOT/pages/connected-apps-developers.adoc]: Good idea trying to smooth out this wording, but I still find it a little hard to parse, so I think you can take it even further. I'll throw a suggestion in the mix, but you can take it or leave it.

```suggestion
This grant type is not recommended, as it exposes user credentials directly to the connected app. It also precludes the use of additional security measures such as multi-factor authentication.
```

File: modules/ROOT/pages/connected-apps-developers.adoc [hunk 3]
```diff
@@ -152,13 +158,14 @@ NOTE: At minimum, `identityProvider_id` and `username` are required for `sub`, a
...
 
 === Client Credentials
 
-The `client_credentials` grant type is suitable for machine-to-machine authentication where a specific user’s permission to access data is not required.
-It uses the `clientId` and `clientSecret` of the client to authorize and access protected data.
+Use the `client_credentials` grant type for machine-to-machine authentication where a specific user’s permission to access data isn't required.
+
+The `client_credentials` grant type uses the `clientId` and `clientSecret` of the client to authorize and access protected data.
+
 CI/CD pipelines using service accounts to programmatically call Anypoint Platform can be configured to use connected apps to prevent service disruptions when configuring xref:multi-factor-authentication.adoc[multi-factor authentication].
```
Comment (dlhforce) [inline modules/ROOT/pages/connected-apps-developers.adoc]: It feels like this belongs with the "Create a Connected App That Acts on Its Own Behalf". Although I can see how it technically belongs under this "Grant Types and Token Flow" section as well. Maybe just add a link between the sections?

File: modules/ROOT/pages/connected-apps-developers.adoc [hunk 4]
```diff
@@ -171,27 +178,27 @@ Most of the scopes in the https://anypoint.mulesoft.com/accounts/api/v2/oauth2/.
 * `openid`: Read-only access to the user's username and unique Anypoint ID.
 * `profile`: Read-only access to the user's Anypoint profile.
 * `email`: Read-only access to the user's email.
-* `offline_access`: Access to the user's data when they are not signed in.
+* `offline_access`: Access to the user's data when they're not signed in.
 * `openid:google_wif`: Enables you to configure Anypoint as an IdP for Google Cloud by providing an `id_token` and an access token during the OAuth2 token flow for `client_credentials` connected apps. This ID token conforms to the IdP requirements as outlined in the https://cloud.google.com/iam/docs/workload-identity-federation-with-other-providers[GCP's WIF requirements].
 
 [NOTE]
-Apps that use `client_credentials` grant type always have the `profile` scope assigned to them implicitly. The data returned by queries using the scope that corresponds to the user who created the connected app.
+Apps that use the `client_credentials` grant type always have the `profile` scope assigned to them implicitly. The data is returned by queries using the scope that corresponds to the user who created the connected app.
```
Comment (dlhforce) [inline modules/ROOT/pages/connected-apps-developers.adoc]: It's a little ambiguous as to how to resolve the grammatical error here, but I read "The data returned by queries" as the subject, and "using [vf. uses]" as the improperly-used verb.

```suggestion
Apps that use the `client_credentials` grant type always have the `profile` scope assigned to them implicitly. The data returned by queries uses the scope that corresponds to the user who created the connected app.
```

File: modules/ROOT/pages/_partials/include-add-scopes.adoc [hunk 5]
```diff
@@ -0,0 +1,10 @@
+//These are the steps for adding scopes
+//Used in connected-apps-end-users
+
+. Click *Add Scopes*.
++
+Scopes are roles with associated permissions, which determine which actions the app can perform within a business group and environment. For more information, see <<endpoint-scopes>>.
```
Comment (hannanelson) [inline modules/ROOT/pages/_partials/include-add-scopes.adoc]:
```suggestion
Scopes are roles with associated permissions, which determine the actions the app can perform within a business group and environment. For more information, see <<endpoint-scopes>>.
```

Comment (hannanelson) [reply modules/ROOT/pages/_partials/include-add-scopes.adoc]: (to avoid which...which)

File: modules/ROOT/pages/_partials/include-add-scopes.adoc [hunk 6]
```diff
@@ -0,0 +1,10 @@
...
+. In Add Scopes, select the scopes to apply and click *Add Scopes*.
```
Comment (hannanelson) [inline modules/ROOT/pages/_partials/include-add-scopes.adoc]:
```suggestion
. In *Add Scopes*, select the scopes to apply and click *Add Scopes*.
```

File: modules/ROOT/pages/_partials/include-create-connected-apps.adoc [hunk 7]
```diff
@@ -0,0 +1,6 @@
+\\Used in creating-connected-apps-dev
+
+. In the Access Management navigation menu, click *Connected Apps*.
+. Optionally, select the business group to create the connected app in from *Business Group*.
```
Comment (hannanelson) [inline modules/ROOT/pages/_partials/include-create-connected-apps.adoc]: You might not need "from Business Group"

File: modules/ROOT/pages/_partials/include-nav-connected-apps.adoc [hunk 8]
```diff
@@ -1 +1,3 @@
-. In the Access Management navigation menu, click *Connected Apps*.
+//Navigation to the Connected Apps page
+
+. In the Access Management navigation menu, click *Connected Apps*.
```
Comment (hannanelson) [inline modules/ROOT/pages/_partials/include-nav-connected-apps.adoc:3]: Could maybe say "Select Access Management > Connected Apps".

File: modules/ROOT/pages/connected-apps-developers.adoc [hunk 9]
```diff
@@ -1,203 +1,34 @@
...
-Access tokens expire after the user’s session expires.
-If the app is using the authorization code grant type, the `offline_access` scope is required to use refresh tokens after the original access tokens expire.
-To request a new access token, you must send a request to the token endpoint using the `refresh_token` grant type.
-When authentication is successful, a refresh token is exchanged for an access token.
-
-=== Password
-
-The `password` grant type is used when the application exchanges the user’s username and password for an access token.
-Because the app has to collect the user’s credentials and its limitation with no mechanisms for things like multi-factor authentication, it is not recommended to use this grant type.
+Organization administrators can enable an allowlist to control which apps can access the organization's users' data.
```
Comment (hannanelson) [inline modules/ROOT/pages/connected-apps-developers.adoc]:
```suggestion
Organization administrators can enable an allowlist to control which apps can access the organization's user data.
```

Comment (hannanelson) [reply modules/ROOT/pages/connected-apps-developers.adoc]: Probably not correct, but the double possessives made my eye twitch.

File: modules/ROOT/pages/connected-apps-developers.adoc [hunk 10]
```diff
@@ -1,203 +1,34 @@
...
-. In the *Connected Apps* page, click the *Scopes Documentation* tab. +
-The *Scopes Documentation* page appears, displaying scopes, their respective permissions, and the products to which they belong.
-+
-. Use the search bar and filters to locate a scope based on the name, display name, permission methods, supported app type, or product label.
-. In the list of scopes, click a scope name to view the associated permissions and learn more about the scope.
-
-You can filter and sort the contents of the *Scopes Documentation* page, and you can also click a scope name to view namespaces, methods, and actions.
-
-== Sign in with Anypoint Platform Widget
+When your app is added to an allowlist, only the set of scopes associated with the app are reviewed and approved by the administrator. If you add new scopes to your app at a later date, you are unable to request those scopes from users until the admin reapproves your app.
```
Comment (hannanelson) [inline modules/ROOT/pages/connected-apps-developers.adoc]:
```suggestion
When your app is added to an allowlist, only the set of scopes associated with the app are reviewed and approved by the administrator. If you add new scopes to your app later, you’re unable to request those scopes from users until the admin reapproves your app.
```

File: modules/ROOT/pages/connected-apps-end-users.adoc [hunk 11]
```diff
@@ -2,27 +2,28 @@
 
 Delegate API access and sign in to third-party applications using your Anypoint Platform credentials. You can also view connected apps created in your root organization or business groups.
 
+[NOTE]
+The organization administrator must set up and integrate the connected app before you can grant access to it. See xref:connected-apps-developers.adoc[]
```
Comment (hannanelson) [inline modules/ROOT/pages/connected-apps-end-users.adoc]:
```suggestion
The organization administrator must set up and integrate the connected app before you can grant access to it. See xref:connected-apps-developers.adoc[].
```

File: modules/ROOT/pages/connected-apps-org-admin.adoc [hunk 12]
```diff
@@ -2,37 +2,84 @@
...
 
-== Owned Apps
+[[connected-app-ownership]]
+== Connected App Ownership
 
-To manage your connected apps, navigate to *Access Management* and select the *Connected Apps* tab. You can add apps in the *Owned Apps* section and manage app access in the *Authorizations* section.
-For more information about the *Owned Apps* section, see the documentation for xref:connected-apps-developers.adoc[app developers].
+Connected app ownership gets transferred to the root organization owner when the user who created the connected app:
+
+* is deleted.
```
Comment (hannanelson) [inline modules/ROOT/pages/connected-apps-org-admin.adoc]: I think these should start with [capital letter](https://confluence.internal.salesforce.com/display/MTDT/MuleSoft+CX+Writing+Style+Reference#:~:text=Capitalize%20the%20first%20letter%20of%20each%20item.) and have no period (even though they complete the intro sentence).

File: modules/ROOT/pages/connected-apps-org-admin.adoc [hunk 13]
```diff
@@ -2,37 +2,84 @@
...
+* is removed from the root organization.
+* loses administrative privileges over the organization.
+
+To maintain security, manually rotate the client secrets of all connected apps in the organization after the transfer. For more information about how to manually rotate client secrets, see <<modify-client-secret>>.
+
+You can also reset client secrets programmatically using the https://anypoint.mulesoft.com/exchange/portals/anypoint-platform/f1e97bc6-315a-4490-82a7-23abe036327a.anypoint-platform/access-management-api/minor/1.0/console/method/%236918/[Access Management API]. 
+
+== Manage Owned Apps
```
Comment (hannanelson) [inline modules/ROOT/pages/connected-apps-org-admin.adoc:18]: Owned > Connected?

File: modules/ROOT/pages/connected-apps-org-admin.adoc [hunk 14]
```diff
@@ -2,37 +2,84 @@
...
 
 To maintain security, you can view and manage connected apps only if you have the Organization Administrator permission in the root organization or in the business group where the connected app resides.
 
-When an organization administrator leaves the organization, is deleted, or is disabled, ownership of their connected apps automatically transfers to the organization owner. To maintain security, manually rotate the client secrets of all connected apps in the organization after the transfer. For more information about how to rotate client secrets, see xref:connected-apps-developers.adoc#modify-the-client-secret-for-a-connected-app[Modify the Client Secret for a Connected App].
+To manage your connected apps:
 
-== Authorizations
+. Navigate to *Access Management* and select the *Connected Apps* tab.
```
Comment (hannanelson) [inline modules/ROOT/pages/connected-apps-org-admin.adoc:24]: Select Access Management > Connected Apps?

File: modules/ROOT/pages/connected-apps-org-admin.adoc [hunk 15]
```diff
@@ -2,37 +2,84 @@
...
+. Make the changes and click *Save changes*.
 
-  Organization members can authorize any connected app to act on their behalf.
+[[modify-client-secret]]
+=== Modify the Client Secret for a Connected App
 
-* Restricted
+include::partial$include-nav-connected-apps.adoc[]
+
+. Click a connected app from the list to enable Edit mode.
```
Comment (hannanelson) [inline modules/ROOT/pages/connected-apps-org-admin.adoc]: should Edit be lc?

File: modules/ROOT/pages/connected-apps-org-admin.adoc [hunk 16]
```diff
@@ -2,37 +2,84 @@
...
+. In the *Secret* field, enter a secret of your choice.
+. Click *Save*.
+
+image::connected-apps-edit-mode.png[Edit mode for a connected app]
++
+[calloutlist]
+. The name and username of the owner of the connected app
```
Comment (hannanelson) [inline modules/ROOT/pages/connected-apps-org-admin.adoc]: Remove the "The" for easier scanning.

File: modules/ROOT/pages/connected-apps-org-admin.adoc [hunk 17]
```diff
@@ -2,37 +2,84 @@
...
+. The name of the connected app
+. The ID of the connected app
+. The client secret of the connected app
+
+=== View Authorizations
+
+. In the *Authorizations* tab, select the app.
+. Click the more actions menu (*...*) for the application and select *Show details*.
```
Comment (hannanelson) [inline modules/ROOT/pages/connected-apps-org-admin.adoc:55]: good place for an icon image...

File: modules/ROOT/pages/connected-apps-org-admin.adoc [hunk 18]
```diff
@@ -2,37 +2,84 @@
...
 
-  Only connected apps that are added to an allowlist by an administrator can be authorized by organization members to act on their behalf.
+=== Revoke Authorization 
+
+. In the *Authorizations* tab, select the app.
+. Click the more actions menu (*...*) for the selected application and select *Remove*.
+. Click *Remove*. 
+
+When you revoke an app’s access, the app’s OAuth 2.0 access tokens and refresh tokens are revoked, and the app can no longer access user data. This action can't be undone.
```
Comment (hannanelson) [inline modules/ROOT/pages/connected-apps-org-admin.adoc]: Should this go before the procedure?

File: modules/ROOT/pages/connected-apps-org-admin.adoc [hunk 19]
```diff
@@ -2,37 +2,84 @@
...
+
+== Configure Authorization Policies
+
+You can configure one of two types of authorization policies for your users:
```
Comment (hannanelson) [inline modules/ROOT/pages/connected-apps-org-admin.adoc]:
```suggestion
You can configure one of these types of authorization policies for your users:
```

File: modules/ROOT/pages/connected-apps-org-admin.adoc [hunk 20]
```diff
@@ -2,37 +2,84 @@
...
+* Unrestricted
++
+Organization members can authorize any connected app to act on their behalf.
+* Restricted
++
+Only connected apps that are added to an allowlist by an administrator are authorized by organization members to act on their behalf.
 
 By default, the **Unrestricted** policy is enabled. If you change the policy to **Restricted**, the application allowlist is enabled, and all existing application authorizations are added automatically.
-When using the **Restricted** policy, you may disable auto-approval of all internal apps, in which case each internal app must be added to the allowlist like any other connected app.
+When using the **Restricted** policy, you can disable auto-approval of all internal apps, in which case you must add each internal app to the <<application-allowlist-admin,application allowlist>> like any other connected app.
```
Comment (hannanelson) [inline modules/ROOT/pages/connected-apps-org-admin.adoc]: Double ** OK?

Comment (hannanelson) [reply modules/ROOT/pages/connected-apps-org-admin.adoc]: Split into two sentences, I think.

File: modules/ROOT/pages/creating-connected-apps-dev.adoc [hunk 21]
```diff
@@ -0,0 +1,195 @@
+= Creating Connected Apps
+
+When you create connected apps, keep in mind: 
+
+* Your organization can own up to 2000 connected apps at the root organization or business group level.
+* Each connected app can have up to 1000 scopes.
+* All scopes in the endpoint map directly to permissions available in Anypoint Platform, except for exceptions noted in <<endpoint-scopes>>. Scopes determine the level of permissions to provide to the app.
```
Comment (hannanelson) [inline modules/ROOT/pages/creating-connected-apps-dev.adoc]: I misread this as "endpoint map" (map as a noun not a verb). Suggest an alternative word like "correspond".

File: modules/ROOT/pages/creating-connected-apps-dev.adoc [hunk 22]
```diff
@@ -0,0 +1,195 @@
+= Creating Connected Apps
+
+When you create connected apps, keep in mind: 
+
+* Your organization can own up to 2000 connected apps at the root organization or business group level.
+* Each connected app can have up to 1000 scopes.
+* All scopes in the endpoint map directly to permissions available in Anypoint Platform, except for exceptions noted in <<endpoint-scopes>>. Scopes determine the level of permissions to provide to the app. 
+* Your Anypoint Platform organization owns your app. Any organization administrator of the root organization or business group in which the app resides can control it.
+* A unique `clientID` is automatically assigned and can’t be changed. 
+* The `clientSecret` has no expiration date by default but you can change it.
```
Comment (hannanelson) [inline modules/ROOT/pages/creating-connected-apps-dev.adoc]: Change the secret or the expiration date?

File: modules/ROOT/pages/creating-connected-apps-dev.adoc [hunk 23]
```diff
@@ -0,0 +1,195 @@
...
+
+== Before You Begin
+
+* You need the Organization Administrator permission at the business group level you want to create connected apps in and specify access scopes for.
```
Comment (hannanelson) [inline modules/ROOT/pages/creating-connected-apps-dev.adoc]: Link to the permissions table?

File: modules/ROOT/pages/creating-connected-apps-dev.adoc [hunk 24]
```diff
@@ -0,0 +1,195 @@
...
+. For *Type*, select *App acts on behalf of a user*. 
+. Select the grant types to use with the app:
+* Authorization Code
+* Password
+* JWT Bearer
++
+Note: Based on the grant type you select, additional information might be requested. For more information about grant types, see <<grant-types-and-token-flow>>.
+. In *Website URL* , enter the URL where users go to learn more about the app.
+. In *Redirect URIs*, configure which URIs users are directed to after authorization. 
+.. Type the URL and click *Add* to add it to the list.
```
Comment (hannanelson) [inline modules/ROOT/pages/creating-connected-apps-dev.adoc]: Since it's just one item in the sublist, make it a hanging paragraph instead.

File: modules/ROOT/pages/creating-connected-apps-dev.adoc [hunk 25]
```diff
@@ -0,0 +1,195 @@
...
+. In *Audience*, select either:
+* *Members of this organization only* if the app is internal to your organization, or
```
Comment (hannanelson) [inline modules/ROOT/pages/creating-connected-apps-dev.adoc]:
```suggestion
* *Members of this organization only* if the app is internal to your organization
```

File: modules/ROOT/pages/creating-connected-apps-dev.adoc [hunk 26]
```diff
@@ -0,0 +1,195 @@
...
+* *All Anypoint Platform users* if it's a third-party public app that can be distributed outside of your organization.
```
Comment (hannanelson) [inline modules/ROOT/pages/creating-connected-apps-dev.adoc]:
```suggestion
* *All Anypoint Platform users* if it's a third-party public app that can be distributed outside of your organization
```

File: modules/ROOT/pages/creating-connected-apps-dev.adoc [hunk 27]
```diff
@@ -0,0 +1,195 @@
...
+include::partial$include-add-scopes.adoc[]
+
+[[create-connected-app-on-its-own-behalf]]
+== Create a Connected App That Acts on Its Own Behalf
+
+include::partial$include-create-connected-apps.adoc[]
+
+. Select *App acts on it's own behalf (client credentials)*.
```
Comment (hannanelson) [inline modules/ROOT/pages/creating-connected-apps-dev.adoc:43]: Argh! Is that really in the UI?

File: modules/ROOT/pages/creating-connected-apps-dev.adoc [hunk 28]
```diff
@@ -0,0 +1,195 @@
...
+=== Authorization Code
+
+Use the `authorization_code` grant type to authorize the app to access a user's data. 
+
+This is an example of the development flow: 
+
+. A developer creates the connected app that acts on behalf of a user with the `authorization_code` grant type.
+. The end user authorizes the app to act on their behalf.
+. The authorization server sends a temporary authorization code back to the specified redirect URI.
+. The app calls the authorization server with the temporary code to receive the access token. +
```
Comment (hannanelson) [inline modules/ROOT/pages/creating-connected-apps-dev.adoc]:
```suggestion
. The app calls the authorization server with the temporary code to receive the access token.
+
```

File: modules/ROOT/pages/creating-connected-apps-dev.adoc [hunk 29]
```diff
@@ -0,0 +1,195 @@
...
+The access token can now be used to call Anypoint Platform APIs on the user’s behalf.
+
+[[refresh-token]]
+=== Refresh Token
+
+Use the `refresh_token` grant type to grant access to user data when the user isn’t signed in. Refresh tokens expire after 90 days. Access tokens expire after the user’s session expires.
+
+To work with OpenID Connect, apps have to exchange refresh tokens for new access tokens after the initial authorization to continue accessing user data.
```
Comment (hannanelson) [inline modules/ROOT/pages/creating-connected-apps-dev.adoc]:
```suggestion
To work with OpenID Connect, apps exchange refresh tokens for new access tokens after the initial authorization to continue accessing user data.
```

File: modules/ROOT/pages/creating-connected-apps-dev.adoc [hunk 30]
```diff
@@ -0,0 +1,195 @@
...
+If the app is using the authorization code grant type, the `offline_access` scope is required to use refresh tokens after the original access tokens expire.
+
+To request a new access token, send a request to the token endpoint using the `refresh_token` grant type. When authentication is successful, a refresh token is exchanged for an access token.
+
+[[password]]
+=== Password
+
+Use the `password` grant type when the application exchanges the user’s username and password for an access token.
+
+[[NOTE]]
```
Comment (hannanelson) [inline modules/ROOT/pages/creating-connected-apps-dev.adoc]: Does this formatting work? [[NOTE]] I usually use just one set of brackets.

File: modules/ROOT/pages/creating-connected-apps-dev.adoc [hunk 31]
```diff
@@ -0,0 +1,195 @@
...
+This grant type is not recommended, as it exposes user credentials directly to the connected app. It also precludes the use of additional security measures such as multi-factor authentication.
```
Comment (hannanelson) [inline modules/ROOT/pages/creating-connected-apps-dev.adoc]:
```suggestion
This grant type isn’t recommended, as it reveals user credentials directly to the connected app. This type also precludes the use of additional security measures, such as multi-factor authentication.
```

File: modules/ROOT/pages/creating-connected-apps-dev.adoc [hunk 32]
```diff
@@ -0,0 +1,195 @@
...
+| `aud`
+| The authentication server. For example, the authentication server for the US region is `https://anypoint.mulesoft.com/accounts/api/v2/oauth2/token`.
+
+|===
+. The app calls the authorization server with the generated assertion.
+
+[[client-credentials]]
+=== Client Credentials
+
+Use the `client_credentials` grant type for machine-to-machine authentication where a specific user’s permission to access data isn't required.
```
Comment (hannanelson) [inline modules/ROOT/pages/creating-connected-apps-dev.adoc]:
```suggestion
Use the `client_credentials` grant type for machine-to-machine authentication when a specific user’s permission to access data isn't required.
```

File: modules/ROOT/pages/creating-connected-apps-dev.adoc [hunk 33]
```diff
@@ -0,0 +1,195 @@
...
+
+The `client_credentials` grant type uses the `clientId` and `clientSecret` of the client to authorize and access protected data.
+
+You can configure CI/CD pipelines using service accounts to programmatically call Anypoint Platform to use connected apps to prevent service disruptions when configuring xref:multi-factor-authentication.adoc[multi-factor authentication].
```
Comment (hannanelson) [inline modules/ROOT/pages/creating-connected-apps-dev.adoc]: Maybe put user value (prevent service disruptions) first?

In any case, this sentence is a long drink of water...

File: modules/ROOT/pages/creating-connected-apps-dev.adoc [hunk 34]
```diff
@@ -0,0 +1,195 @@
...
+Read-only access to the user's Anypoint Platform profile.
+* `email`
++
+Read-only access to the user's email.
+* `offline_access`
++
+Access to the user's data when they're not signed in.
+* `openid:google_wif`
++
+Enables you to configure Anypoint Platform as an IdP for Google Cloud by providing an `id_token` and an access token during the OAuth2 token flow for `client_credentials` connected apps. This ID token conforms to the IdP requirements as outlined in the https://cloud.google.com/iam/docs/workload-identity-federation-with-other-providers[GCP's WIF requirements].
```
Comment (hannanelson) [inline modules/ROOT/pages/creating-connected-apps-dev.adoc]: Not parallel with the other bullets.

File: modules/ROOT/pages/creating-connected-apps-dev.adoc [hunk 35]
```diff
@@ -0,0 +1,195 @@
...
+
+[NOTE]
+Apps that use the `client_credentials` grant type always have the `profile` scope assigned to them implicitly. The data returned by queries uses the scope that corresponds to the user who created the connected app.
+
+[[scopes-documentation]]
+=== Access Scopes Documentation
+
+For more information on scopes and permissions in the Connected Apps feature, see the scopes documentation page in Access Management:
```
Comment (hannanelson) [inline modules/ROOT/pages/creating-connected-apps-dev.adoc]:
```suggestion
For more information about scopes and permissions in the Connected Apps feature, see the scopes documentation page in Access Management:
```

File: modules/ROOT/pages/creating-connected-apps-dev.adoc [hunk 36]
```diff
@@ -0,0 +1,195 @@
...
+
+include::partial$include-nav-connected-apps.adoc[]
+
+. In *Connected Apps*, click the *Scopes Documentation* tab. 
++
+The *Scopes Documentation* page appears, showing the scopes, their respective permissions, and the products to which they belong.
```
Comment (hannanelson) [inline modules/ROOT/pages/creating-connected-apps-dev.adoc]:
```suggestion
*Scopes Documentation* shows the scopes, their respective permissions, and the products they belong to.
```

File: modules/ROOT/pages/_partials/include-nav-connected-apps.adoc (hunk 8 above)
Comment (valkyrie69) [reply modules/ROOT/pages/_partials/include-nav-connected-apps.adoc:3]: They're already in Access Management, so there's no Access Management to select.

File: modules/ROOT/pages/connected-apps-org-admin.adoc (hunk 13 above)
Comment (valkyrie69) [reply modules/ROOT/pages/connected-apps-org-admin.adoc:18]: It's Owned.

File: modules/ROOT/pages/connected-apps-org-admin.adoc (hunk 20 above)
Comment (valkyrie69) [reply modules/ROOT/pages/connected-apps-org-admin.adoc]: The ** is markdown and renders correctly. This was prob written by a dev originally.

File: modules/ROOT/pages/creating-connected-apps-dev.adoc (hunk 27 above)
Comment (valkyrie69) [reply modules/ROOT/pages/creating-connected-apps-dev.adoc:43]: Yep. *sadface*

File: modules/ROOT/pages/creating-connected-apps-dev.adoc (hunk 30 above)
Comment (valkyrie69) [reply modules/ROOT/pages/creating-connected-apps-dev.adoc]: Auto pilot error

File: modules/ROOT/pages/connected-apps-org-admin.adoc (hunk 17 above)
Comment (valkyrie69) [reply modules/ROOT/pages/connected-apps-org-admin.adoc:55]: Good idea. I'll add this icon later on. I think there are many occurrences of the ... in AM.

**Review by hannanelson — APPROVED** [review]
